types/node/index.d.ts

```diff
@@ -756,20 +756,23 @@ declare module "http" {
         write(str: string, encoding?: string, cb?: Function): boolean;
         write(str: string, encoding?: string, fd?: string): boolean;
 
-        writeContinue(): void;
-        writeHead(statusCode: number, reasonPhrase?: string, headers?: any): void;
-        writeHead(statusCode: number, headers?: any): void;
-        statusCode: number;
-        statusMessage: string;
+        addTrailers(headers: any): void;
+        finished: boolean;
+        getHeader(name: string): string;
+        getHeaderNames(): string[]
+        getHeaders(): any
```

Comment:
I don't agree here (but maybe I'm wrong). To my understanding Typescript checks types on their shape (see http://www.typescriptlang.org/docs/handbook/interfaces.html) so if an interface tells it has a property foo of type bar then everything which has a property foo of type bar matches.